configdb/hardware/models.py

```diff
@@ -222,6 +222,11 @@ class InstrumentType(BaseModel):
         validators=[MinValueValidator(0)],
         help_text='The default exposure time to use for acquisition exposures with this instrument type.'
     )
+    minimum_flat_exposure_time = models.FloatField(
```

Comment:
should it be `default_flat_exposure_time` since it is setting the exposure_time value for flats?